sentences/fr/homeassistant_HassStartTimer.yaml

```
@@ -77,25 +77,25 @@ intents:
           # Mets un minuteur Pizza pour 5 minute
           - "<mets> un <minuteur> {timer_name:name} pour <timer_duration>"
           # Crée un minuteur de 5 minute appelé Pizza
-          - "<cree> un <minuteur> de <timer_duration> <appele> {timer_name:name}"
+          - "<cree> un <minuteur> de <timer_duration> <appele> [<le>] {timer_name:name}"
           # Crée un minuteur d'une heure appelé Pizza
-          - "<cree> un <minuteur> d'<timer_duration> <appele> {timer_name:name}"
+          - "<cree> un <minuteur> d'<timer_duration> <appele> [<le>] {timer_name:name}"
           # Crée un minuteur pour 5 minute appelé Pizza
-          - "<cree> un <minuteur> pour <timer_duration> <appele> {timer_name:name}"
+          - "<cree> un <minuteur> pour <timer_duration> <appele> [<le>] {timer_name:name}"
           # Démarre un minuteur de 5 minute appelé Pizza
-          - "<active> un <minuteur> de <timer_duration> <appele> {timer_name:name}"
+          - "<active> un <minuteur> de <timer_duration> <appele> [<le>] {timer_name:name}"
           # Démarre un minuteur d'une heure appelé Pizza
-          - "<active> un <minuteur> d'<timer_duration> <appele> {timer_name:name}"
+          - "<active> un <minuteur> d'<timer_duration> <appele> [<le>] {timer_name:name}"
           # Démarre un minuteur pour 5 minute appelé Pizza
-          - "<active> un <minuteur> pour <timer_duration> <appele> {timer_name:name}"
+          - "<active> un <minuteur> pour <timer_duration> <appele> [<le>] {timer_name:name}"
           # Mets un minuteur de 5 minute appelé Pizza
-          - "<mets> un <minuteur> de <timer_duration> <appele> {timer_name:name}"
+          - "<mets> un <minuteur> de <timer_duration> <appele> [<le>] {timer_name:name}"
           # Mets un minuteur d'une heure appelé Pizza
-          - "<mets> un <minuteur> d'<timer_duration> <appele> {timer_name:name}"
+          - "<mets> un <minuteur> d'<timer_duration> <appele> [<le>] {timer_name:name}"
           # Mets un minuteur pour 5 minute appelé Pizza
-          - "<mets> un <minuteur> pour <timer_duration> <appele> {timer_name:name}"
+          - "<mets> un <minuteur> pour <timer_duration> <appele> [<le>] {timer_name:name}"
         expansion_rules:
-          appele: "(appelé|nommé|surnomé)"
+          appele: "(appelé|nommé|surnommé|pour)"
```

```suggestion
          appele: "(appelé|nommé|surnommé|pour [<le>])"
```
I wonder if we should just use this expansion rule as "Met un minuteur appelé la pizza" doesn't make sense.

```

---

I like the idea of limiting the risk of regression for my own use case. I'll try and test on my side.

---

Agreed.